dolphinscheduler-python/pydolphinscheduler/README.md

```diff
@@ -84,4 +84,4 @@ If you are interested in how to release **PyDolphinScheduler**, you could go and
 
 ## What's more
 
-For more detail information, please go to see **PyDolphinScheduler** [document](https://dolphinscheduler.apache.org/python/index.html)
+For more detail information, please go to see **PyDolphinScheduler** [document](https://dolphinscheduler.apache.org/python/3.0.0/index.html)
```

Comment:
Good catch, I also add in https://github.com/apache/dolphinscheduler/pull/11170/files#diff-b26d7dd07b6c97abef7a204c696762ad9e04fc24713c3817d6ecb22fb84986f3

Reply:
Thx for the information. I will change `3.0.0` to `dev` or you could merge your PR and I will rebase on it.